OsmAnd/res/values/strings.xml

```diff
@@ -1112,11 +1112,11 @@
 	and more…
     </string -->
     <string name="rendering_attr_contourColorScheme_description">Contour lines color scheme</string>
-    <string name="save_track_min_speed">Logging minimum speed</string>
+    <string name="save_track_min_speed">Minimum logging speed</string>
```

Comment:
No. ;)

"Logging minimum speed" has a different meaning from "Minimum logging speed". (There is no such thing as "logging speed" similarly in your changes below.)

Reply:
It has me confused. In my mind these are the options.

**Logging minimum speed**: Starts with something that could be felling trees. Means something is logging the minimum speed, as an action.
**Minimum logging speed**: The slowest speed that is logged (and I thought this to be the intention)
**Logging speed minimum**: The slowest speed logging can be done at, as a convoluted way of saying option 2.

Reply:
We could work with 'threshold', but asking my colleagues here that would lead to lengthy translations in some languages, so I'd like to stick to 'min something'.
How about:
Minimum speed for logging
Minimum accuracy for logging
Minimum displacement for logging